### PR TITLE
Faded first line of composer

### LINKED ISSUE #17394: Faded first line of text in composer (Android)
paste a long message, collapse the composer, send the message, and start typing a new one

**Steps:**

1. Paste a long message into composer
2. Collapse composer
3. Send the message
4. Start typing new message
5. Pay attention at first line of the text

**Actual result:** text is faded
photo_2023-09-22 17 27 48

https://github.com/status-im/status-mobile/assets/97245802/10da0345-d323-405f-a19b-ceb9e94555d2

### Additional Information

- Status version: nightly 
- Operating System: Android

[comment]: # (Please, add logs/notes if necessary)

### PR DESCRIPTION
**Pull request by ibrkhalil**
fixes #17394 

### Summary

Fixes a UI issue relate to top gradient in composer

Screenshots not applicable

status: ready

### CODE REVIEW

File: src/status_im2/contexts/chat/composer/handlers.cljs [hunk 1]
```diff
@@ -27,9 +27,6 @@
   (when (> (reanimated/get-shared-value last-height) (* constants/background-threshold max-height))
     (reanimated/animate opacity 1)
     (reanimated/set-shared-value background-y 0))
-  (when (= @gradient-z-index -1)
-    (reanimated/animate gradient-opacity 1)
-    (reset! gradient-z-index 1))
```
Comment (OmarBasem) [inline src/status_im2/contexts/chat/composer/handlers.cljs:32]: I am not sure this is a correct change. What would happen if there is a long text, and cursor is at the end, then we blur, and we try to focus, is the gradient at the top going to be present?

Comment (ibrkhalil) [reply src/status_im2/contexts/chat/composer/handlers.cljs:32]: Screencast from 04 أكت, 2023 EET 10_39_31 ص


Here's how it looks Omar, I tested it and it works as expected.

Comment (OmarBasem) [reply src/status_im2/contexts/chat/composer/handlers.cljs:32]: @ibrkhalil Have you tested on both iOS and Android?

Comment (ibrkhalil) [reply src/status_im2/contexts/chat/composer/handlers.cljs:32]: Yes

Comment (ibrkhalil) [reply src/status_im2/contexts/chat/composer/handlers.cljs:32]: And I think QA will test it better than me to be honest :)

**Review by OmarBasem — APPROVED** [review]

**Comment by pavloburykh** [comment]
@ibrkhalil thanx for the fix. Ready for merge.

**Pull request merged**